src/content/cm-addons/lint-metadata.js

```diff
@@ -15,21 +16,21 @@ function lintMetadata(text, options) {
   for (let [i, metaLine] of meta.entries()) {
     try {
       // Only consider meta lines.
-      if (metaLine.substr(0, 4) == '// @') {
+      if (metaLine.substr(0, META_POSN) === META_PREFIX) {
         parseMetaLine(metaLine.trim());
       }
     } catch (e) {
       let ann = {
-        severity: 'warning',
-        from: { line: metaStartLine + i, ch: e.location.start.column - 1 },
-        to: { line: metaStartLine + i, ch: e.location.end.column - 1 },
-        message: e.message,
+        'severity': 'warning',
+        'from': { line: metaStartLine + i, ch: e.location.start.column - 1 },
+        'to': { line: metaStartLine + i, ch: e.location.end.column - 1 },
+        'message': e.message
       };
       // Turn "Expected...but 'w' found" into "Expected...but 'word' found".
-      if (e.message.match(/Expected.*but.*found/) && ann.from.ch == META_POSN) {
-        let m = metaLine.match(/^\/\/ @([^\s]*)/);
+      if (e.message.match(/Expected .* but .* found\./) && ann.from.ch === META_POSN) {
+        let m = metaLine.substr(META_POSN).match(/^(\s+|\S*)/);
+        ann.message = ann.message.replace(/( but ")([^"]*)(" found\.)$/, `$1${m[1]}$3`);
```

Comment:
Line break here, 80 character max.

Reply:
Oops, trying to respect 80 chars, sometimes it fails...